ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master_metal.yaml

```diff
@@ -0,0 +1,58 @@
+build_root:
+  project_image:
+    dockerfile_literal: |
+      FROM registry.ci.openshift.org/ocp/ubi-python-311:9
+      USER 0
+releases:
+  arm64-latest:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
+      version: "4.18"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.18"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: e2e-rds-core-single-node
+  capabilities:
+  - intranet
+  restrict_network_access: false
+  steps:
+    cluster_profile: metal-perfscale-jetlag
+    env:
+      BAREMETAL: "true"
+      BASTION_CP_INTERFACE: ens7f1
+      CRUCIBLE: "false"
+      ISOLATED_CORES: 2-39,42-79
+      JETLAG_BRANCH: main
```

Comment:
Isn't better to point to a specific release?

Reply:
I would want to stick to main branch all the time with an intention to stay updated about any changes and failures too rather than staying outdated on a specific release.

Reply:
you can use **_JETLAG_LATEST=true_** as well